core/src/main/java/com/sequenceiq/cloudbreak/service/parcel/ParcelService.java

```diff
@@ -91,7 +91,7 @@ public ParcelOperationStatus removeUnusedParcelComponents(Stack stack) throws Cl
 
     public ParcelOperationStatus removeUnusedParcelComponents(Stack stack, Set<ClusterComponent> clusterComponentsByBlueprint) throws CloudbreakException {
         LOGGER.debug("Starting to remove unused parcels from the cluster.");
-        Set<String> parcelsFromImage = imageReaderService.getParcelNames(stack.getId(), stack.isDatalake());
+        Set<String> parcelsFromImage = imageReaderService.getParcelNames(stack.getId(), false);
```

Comment:
why is this always `false`? And if so then the parameter itself is not needed, as it is always false anyways

Reply:
I changed it because we're using the service names to determine the custom parcels in the cluster. In the case of DL this method returned only with the CDH parcel and the parcel removal logic believed that all other pre-warmed parcels are custom parcels. 

I'm removing this parameter.